docs/signature_specification.md

```diff
@@ -0,0 +1,96 @@
+# Overview
+
+This specification covers the generation of file fingerprints. A file fingerprint is a json document with various data extracted from the file, to facilitate security and compliance scanning through SAPI. 
+
+A file fingerprint contains the following data: 
+- path 
+- full file hash(es) 
+- partial file hashes (not covered yet) 
+- licences (not covered yet) 
+- full file contents of the file (for manifest files, not covered yet) 
+
+# Example Fingerprint
+
+The following fingerprint contains a path, two format 1 hashes ("double hash"), and one format 3 hash ("uhash"):
+
+```
```

Comment:
Just an FYI, you can get syntax highlighting in GitHub Markdown by specifying a language after the backticks i.e.

\```json
{}
\```